src/activities/assignments/AssignmentEntity.js

```diff
@@ -829,8 +829,8 @@ export class AssignmentEntity extends Entity {
 		}
 
 		if (typeof assignment.completionType !== 'undefined' &&
-				assignment.completionType !== this.completionType() &&
-				this.canEditCompletionType()) {
+			(shouldSaveSubmissionType || assignment.completionType !== this.completionTypeValue()) &&
```

Comment:
If `submissionType` is included in the PATCH, we must include the `completionType` (even if it hasn't changed), otherwise it will be reset to `onSubmission`.
https://search.d2l.dev/xref/lms/le/dropbox/D2L.LE.Dropbox.Extensibility/Service/Default/DropboxService.cs?r=a46e549b#1149

Reply:
Is it possible for the user to have `canEditSubmissionType` but not have `canEditCompletionType`?

Reply:
Not possible with the way we are returning the siren entity. Either both actions are included or they're both not included.

https://search.d2l.dev/xref/lms/le/dropbox/D2L.LE.Dropbox.Extensibility/HM/Folders/Serializers/Siren/SirenFoldersSerializer.cs?r=f4e056be#1261-1277